payara-embedded-web: java.lang.NoClassDefFoundError: fish/payara/nucleus/requesttracing/RequestTracingService

Hello,

I've been using payara-embedded-web 4.1.152.1 for a long time, since the payara-micro take too much time(about 350ms) to startup because I'm running it on an embedded arm like a beagleBone and it takes too much time to extract war files, so I'm using the payara-embedded-web deploying applications directly from directories(about 130ms).
Today I decided to update the payara-embedded-web and use the latest version(4.1.1.163 and 4.1.1.162), but it seems to have a dependency issue.

OBS: version 4.1.1.161.1 is working fine.

**The exception:**
```
GRAVE: Shutting down server due to startup exception
java.lang.NoClassDefFoundError: fish/payara/nucleus/requesttracing/RequestTracingService
        at com.sun.enterprise.v3.services.impl.ContainerMapper.<init>(ContainerMapper.java:107)
        at com.sun.enterprise.v3.services.impl.GlassfishNetworkListener.configureHttpProtocol(GlassfishN                                                     etworkListener.java:166)
        at org.glassfish.grizzly.config.GenericGrizzlyListener.configureSubProtocol(GenericGrizzlyListen                                                     er.java:420)
        at org.glassfish.grizzly.config.GenericGrizzlyListener.configureProtocol(GenericGrizzlyListener.                                                     java:409)
        at org.glassfish.grizzly.config.GenericGrizzlyListener.configure(GenericGrizzlyListener.java:305                                                     )
        at com.sun.enterprise.v3.services.impl.GrizzlyProxy.initialize(GrizzlyProxy.java:125)
        at com.sun.enterprise.v3.services.impl.GrizzlyService.createNetworkProxy(GrizzlyService.java:539                                                     )
        at com.sun.enterprise.v3.services.impl.GrizzlyService.postConstruct(GrizzlyService.java:490)
        at org.jvnet.hk2.internal.ClazzCreator.postConstructMe(ClazzCreator.java:326)
        at org.jvnet.hk2.internal.ClazzCreator.create(ClazzCreator.java:374)
        at org.jvnet.hk2.internal.SystemDescriptor.create(SystemDescriptor.java:471)
        at org.glassfish.hk2.runlevel.internal.AsyncRunLevelContext.findOrCreate(AsyncRunLevelContext.ja                                                     va:228)
        at org.glassfish.hk2.runlevel.RunLevelContext.findOrCreate(RunLevelContext.java:85)
        at org.jvnet.hk2.internal.Utilities.createService(Utilities.java:2022)
        at org.jvnet.hk2.internal.ServiceHandleImpl.getService(ServiceHandleImpl.java:114)
        at org.jvnet.hk2.internal.ServiceHandleImpl.getService(ServiceHandleImpl.java:88)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.oneJob(CurrentTaskFuture.ja                                                     va:1213)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.run(CurrentTaskFuture.java:                                                     1144)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$UpOneLevel.run(CurrentTaskFuture.java:7                                                     62)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$UpAllTheWay.go(CurrentTaskFuture.java:5                                                     61)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$UpAllTheWay.access$100(CurrentTaskFutur                                                     e.java:443)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture.go(CurrentTaskFuture.java:154)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture.go(CurrentTaskFuture.java:149)
        at org.glassfish.hk2.runlevel.internal.AsyncRunLevelContext.proceedTo(AsyncRunLevelContext.java:                                                     436)
        at org.glassfish.hk2.runlevel.internal.RunLevelControllerImpl.proceedTo(RunLevelControllerImpl.j                                                     ava:71)
        at com.sun.enterprise.v3.server.AppServerStartup.proceedTo(AppServerStartup.java:534)
        at com.sun.enterprise.v3.server.AppServerStartup.run(AppServerStartup.java:331)
        at com.sun.enterprise.v3.server.AppServerStartup.doStart(AppServerStartup.java:228)
        at com.sun.enterprise.v3.server.AppServerStartup.start(AppServerStartup.java:219)
        at com.sun.enterprise.glassfish.bootstrap.GlassFishImpl.start(GlassFishImpl.java:79)
        at br.com.bematech.hubprinter.bootstrap.Server.main(Server.java:124)
Caused by: java.lang.ClassNotFoundException: fish.payara.nucleus.requesttracing.RequestTracingService
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 31 more
```


**My pom.xml:**
```
	<dependencies>
		<!-- Payara Embedded Web --> 
		<dependency>
    		<groupId>fish.payara.extras</groupId>
    		<artifactId>payara-embedded-web</artifactId>
    		<!-- <version>4.1.152.1</version> --> 
    		<version>4.1.1.161.1</version>
    		<type>jar</type>
		</dependency>
	

		<!-- log4j -->
		<dependency>
			<groupId>log4j</groupId>
			<artifactId>log4j</artifactId>
			<version>1.2.16</version>
			<type>jar</type>
		</dependency>

	</dependencies>
```


**The initialization console:**
```
ago 23, 2016 6:19:32 PM org.jvnet.hk2.config.ConfigParser handleElement
GRAVE: Ignoring unrecognized element jms-service at Line number = 101
Column number = 72
System Id = null
Public Id = null
Location Uri= null
CharacterOffset = 5783

ago 23, 2016 6:19:36 PM org.glassfish.security.services.impl.authorization.AuthorizationServiceImpl init                                                     ialize
INFORMAÇÕES: Authorization Service has successfully initialized.
log4j:WARN No appenders could be found for logger (org.jboss.logging).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
ago 23, 2016 6:19:45 PM com.sun.enterprise.v3.services.impl.GrizzlyService createNetworkProxy
INFORMAÇÕES: Network listener https-listener on port 0 disabled per domain.xml
ago 23, 2016 6:19:45 PM com.sun.enterprise.v3.admin.adapter.AdminEndpointDecider setGuiContextRoot
INFORMAÇÕES: Admin Console Adapter: context root: /admin
ago 23, 2016 6:19:45 PM com.sun.enterprise.v3.admin.adapter.AdminEndpointDecider setGuiContextRoot
INFORMAÇÕES: Admin Console Adapter: context root: /admin
ago 23, 2016 6:19:45 PM com.sun.enterprise.v3.admin.adapter.AdminEndpointDecider setGuiContextRoot
INFORMAÇÕES: Admin Console Adapter: context root: /admin
ago 23, 2016 6:19:51 PM com.sun.enterprise.v3.server.AppServerStartup postStartupJob
INFORMAÇÕES: Undefined Product Name - define product and version info in config/branding  0.0.0.0.0  (0)                                                      startup time : Embedded (13.153ms), startup services(10.404ms), total(23.557ms)
ago 23, 2016 6:19:51 PM com.sun.enterprise.v3.server.AppServerStartup postStartupJob
GRAVE: Shutting down server due to startup exception
java.lang.NoClassDefFoundError: fish/payara/nucleus/requesttracing/RequestTracingService
        at com.sun.enterprise.v3.services.impl.ContainerMapper.<init>(ContainerMapper.java:107)
        at com.sun.enterprise.v3.services.impl.GlassfishNetworkListener.configureHttpProtocol(GlassfishN                                                     etworkListener.java:166)
        at org.glassfish.grizzly.config.GenericGrizzlyListener.configureSubProtocol(GenericGrizzlyListen                                                     er.java:420)
        at org.glassfish.grizzly.config.GenericGrizzlyListener.configureProtocol(GenericGrizzlyListener.                                                     java:409)
        at org.glassfish.grizzly.config.GenericGrizzlyListener.configure(GenericGrizzlyListener.java:305                                                     )
        at com.sun.enterprise.v3.services.impl.GrizzlyProxy.initialize(GrizzlyProxy.java:125)
        at com.sun.enterprise.v3.services.impl.GrizzlyService.createNetworkProxy(GrizzlyService.java:539                                                     )
        at com.sun.enterprise.v3.services.impl.GrizzlyService.postConstruct(GrizzlyService.java:490)
        at org.jvnet.hk2.internal.ClazzCreator.postConstructMe(ClazzCreator.java:326)
        at org.jvnet.hk2.internal.ClazzCreator.create(ClazzCreator.java:374)
        at org.jvnet.hk2.internal.SystemDescriptor.create(SystemDescriptor.java:471)
        at org.glassfish.hk2.runlevel.internal.AsyncRunLevelContext.findOrCreate(AsyncRunLevelContext.ja                                                     va:228)
        at org.glassfish.hk2.runlevel.RunLevelContext.findOrCreate(RunLevelContext.java:85)
        at org.jvnet.hk2.internal.Utilities.createService(Utilities.java:2022)
        at org.jvnet.hk2.internal.ServiceHandleImpl.getService(ServiceHandleImpl.java:114)
        at org.jvnet.hk2.internal.ServiceHandleImpl.getService(ServiceHandleImpl.java:88)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.oneJob(CurrentTaskFuture.ja                                                     va:1213)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.run(CurrentTaskFuture.java:                                                     1144)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$UpOneLevel.run(CurrentTaskFuture.java:7                                                     62)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$UpAllTheWay.go(CurrentTaskFuture.java:5                                                     61)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$UpAllTheWay.access$100(CurrentTaskFutur                                                     e.java:443)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture.go(CurrentTaskFuture.java:154)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture.go(CurrentTaskFuture.java:149)
        at org.glassfish.hk2.runlevel.internal.AsyncRunLevelContext.proceedTo(AsyncRunLevelContext.java:                                                     436)
        at org.glassfish.hk2.runlevel.internal.RunLevelControllerImpl.proceedTo(RunLevelControllerImpl.j                                                     ava:71)
        at com.sun.enterprise.v3.server.AppServerStartup.proceedTo(AppServerStartup.java:534)
        at com.sun.enterprise.v3.server.AppServerStartup.run(AppServerStartup.java:331)
        at com.sun.enterprise.v3.server.AppServerStartup.doStart(AppServerStartup.java:228)
        at com.sun.enterprise.v3.server.AppServerStartup.start(AppServerStartup.java:219)
        at com.sun.enterprise.glassfish.bootstrap.GlassFishImpl.start(GlassFishImpl.java:79)
        at br.com.bematech.hubprinter.bootstrap.Server.main(Server.java:124)
Caused by: java.lang.ClassNotFoundException: fish.payara.nucleus.requesttracing.RequestTracingService
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 31 more

ago 23, 2016 6:19:51 PM com.sun.enterprise.v3.admin.StopServer doExecute
INFORMAÇÕES: Server shutdown initiated
23-08-2016 18:19:51,711  INFO [main] (Server.java:106) - Deploying application
ago 23, 2016 6:20:02 PM com.sun.enterprise.security.SecurityLifecycle <init>
INFORMAÇÕES: Java security manager is disabled.
ago 23, 2016 6:20:02 PM com.sun.enterprise.security.SecurityLifecycle onInitialization
INFORMAÇÕES: Entering Security Startup Service.
ago 23, 2016 6:20:02 PM com.sun.enterprise.security.PolicyLoader loadPolicy
INFORMAÇÕES: Loading policy provider com.sun.enterprise.security.jacc.provider.SimplePolicyProvider.
ago 23, 2016 6:20:02 PM com.sun.enterprise.security.auth.realm.Realm doInstantiate
INFORMAÇÕES: Realm [admin-realm] of classtype [com.sun.enterprise.security.auth.realm.file.FileRealm] su                                                     ccessfully created.
ago 23, 2016 6:20:02 PM com.sun.enterprise.security.auth.realm.Realm doInstantiate
INFORMAÇÕES: Realm [file] of classtype [com.sun.enterprise.security.auth.realm.file.FileRealm] successfu                                                     lly created.
ago 23, 2016 6:20:02 PM com.sun.enterprise.security.auth.realm.Realm doInstantiate
INFORMAÇÕES: Realm [certificate] of classtype [com.sun.enterprise.security.auth.realm.certificate.Certif                                                     icateRealm] successfully created.
ago 23, 2016 6:20:02 PM com.sun.enterprise.security.SecurityLifecycle onInitialization
INFORMAÇÕES: Security Service(s) started successfully.
ago 23, 2016 6:20:04 PM com.sun.enterprise.web.WebContainer createHttpListener
INFORMAÇÕES: Created HTTP listener http-listener on host/port 0.0.0.0:80
ago 23, 2016 6:20:05 PM com.sun.enterprise.web.VirtualServer addProbes
GRAVE: Error adding HttpProbes. NetworkListener http-listeners GrizzlyProxy is NULL
ago 23, 2016 6:20:05 PM com.sun.enterprise.web.VirtualServer addProbes
GRAVE: Error adding HttpProbes. NetworkListener https-listeners GrizzlyProxy is NULL
ago 23, 2016 6:20:05 PM com.sun.enterprise.web.WebContainer createHosts
INFORMAÇÕES: Created virtual server server
ago 23, 2016 6:20:08 PM org.apache.catalina.realm.JAASRealm setContainer
INFORMAÇÕES: Setting JAAS app name glassfish-web
ago 23, 2016 6:20:08 PM com.sun.enterprise.web.WebContainer loadSystemDefaultWebModules
INFORMAÇÕES: Virtual server server loaded default web module
ago 23, 2016 6:20:09 PM org.apache.catalina.core.ContainerBase startChildren
GRAVE: Container StandardEngine[glassfish-web].StandardHost[server].StandardContext[] has not been start                                                     ed
org.apache.catalina.LifecycleException: org.apache.catalina.LifecycleException: java.lang.NoClassDefFoun                                                     dError: fish/payara/nucleus/requesttracing/RequestTracingService
        at org.apache.catalina.core.StandardContext.start(StandardContext.java:5926)
        at com.sun.enterprise.web.WebModule.start(WebModule.java:692)
        at org.apache.catalina.core.ContainerBase.startChildren(ContainerBase.java:1593)
        at org.apache.catalina.core.ContainerBase.start(ContainerBase.java:1290)
        at org.apache.catalina.core.StandardHost.start(StandardHost.java:982)
        at org.apache.catalina.core.ContainerBase.startChildren(ContainerBase.java:1593)
        at org.apache.catalina.core.ContainerBase.start(ContainerBase.java:1290)
        at org.apache.catalina.core.StandardEngine.start(StandardEngine.java:402)
        at org.apache.catalina.startup.Embedded.start(Embedded.java:995)
        at com.sun.enterprise.web.WebContainer.postConstruct(WebContainer.java:867)
        at org.jvnet.hk2.internal.ClazzCreator.postConstructMe(ClazzCreator.java:326)
        at org.jvnet.hk2.internal.ClazzCreator.create(ClazzCreator.java:374)
        at org.jvnet.hk2.internal.SystemDescriptor.create(SystemDescriptor.java:471)
        at org.jvnet.hk2.internal.SingletonContext$1.compute(SingletonContext.java:83)
        at org.jvnet.hk2.internal.SingletonContext$1.compute(SingletonContext.java:71)
        at org.glassfish.hk2.utilities.cache.Cache$OriginThreadAwareFuture$1.call(Cache.java:97)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at org.glassfish.hk2.utilities.cache.Cache$OriginThreadAwareFuture.run(Cache.java:154)
        at org.glassfish.hk2.utilities.cache.Cache.compute(Cache.java:199)
        at org.jvnet.hk2.internal.SingletonContext.findOrCreate(SingletonContext.java:122)
        at org.jvnet.hk2.internal.Utilities.createService(Utilities.java:2022)
        at org.jvnet.hk2.internal.ServiceHandleImpl.getService(ServiceHandleImpl.java:114)
        at org.jvnet.hk2.internal.ServiceHandleImpl.getService(ServiceHandleImpl.java:88)
        at org.glassfish.internal.data.EngineInfo.getContainer(EngineInfo.java:86)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.startContainers(ApplicationLifecycle.java:1                                                     012)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.setupContainerInfos(ApplicationLifecycle.ja                                                     va:718)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:379)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:220)
        at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:487)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:539)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:535)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:534)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:565)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:557)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:556)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1464)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:109)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1846)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1722)
        at com.sun.enterprise.admin.cli.embeddable.DeployerImpl.deploy(DeployerImpl.java:134)
        at br.com.bematech.hubprinter.bootstrap.Server.deploy(Server.java:107)
        at br.com.bematech.hubprinter.bootstrap.Server.main(Server.java:127)
Caused by: org.apache.catalina.LifecycleException: java.lang.NoClassDefFoundError: fish/payara/nucleus/r                                                     equesttracing/RequestTracingService
        at com.sun.enterprise.web.WebModuleContextConfig.start(WebModuleContextConfig.java:279)
        at org.apache.catalina.startup.ContextConfig.lifecycleEvent(ContextConfig.java:486)
        at org.apache.catalina.util.LifecycleSupport.fireLifecycleEvent(LifecycleSupport.java:163)
        at org.apache.catalina.core.StandardContext.start(StandardContext.java:5923)
        ... 45 more
Caused by: java.lang.NoClassDefFoundError: fish/payara/nucleus/requesttracing/RequestTracingService
        at org.apache.catalina.core.StandardWrapper.<init>(StandardWrapper.java:225)
        at org.apache.catalina.core.StandardContext.createWrapper(StandardContext.java:4220)
        at com.sun.enterprise.web.pwc.PwcWebModule.createWrapper(PwcWebModule.java:259)
        at com.sun.enterprise.web.TomcatDeploymentConfig.configureStandardContext(TomcatDeploymentConfig                                                     .java:402)
        at com.sun.enterprise.web.TomcatDeploymentConfig.configureWebModule(TomcatDeploymentConfig.java:                                                     108)
        at com.sun.enterprise.web.WebModuleContextConfig.start(WebModuleContextConfig.java:264)
        ... 48 more
Caused by: java.lang.ClassNotFoundException: fish.payara.nucleus.requesttracing.RequestTracingService
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 54 more

ago 23, 2016 6:20:14 PM org.glassfish.internal.data.ModuleInfo load
GRAVE: Exception while invoking class org.glassfish.ejb.startup.EjbDeployer load method
java.lang.RuntimeException: EJB Container initialization error
        at org.glassfish.ejb.startup.EjbApplication.loadContainers(EjbApplication.java:237)
        at org.glassfish.ejb.startup.EjbDeployer.load(EjbDeployer.java:290)
        at org.glassfish.ejb.startup.EjbDeployer.load(EjbDeployer.java:100)
        at org.glassfish.internal.data.ModuleInfo.load(ModuleInfo.java:206)
        at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:314)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:497)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:220)
        at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:487)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:539)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:535)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:534)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:565)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:557)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:556)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1464)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:109)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1846)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1722)
        at com.sun.enterprise.admin.cli.embeddable.DeployerImpl.deploy(DeployerImpl.java:134)
        at br.com.bematech.hubprinter.bootstrap.Server.deploy(Server.java:107)
        at br.com.bematech.hubprinter.bootstrap.Server.main(Server.java:127)
Caused by: java.lang.NoClassDefFoundError: fish/payara/nucleus/requesttracing/RequestTracingService
        at com.sun.ejb.containers.BaseContainer.<init>(BaseContainer.java:537)
        at com.sun.ejb.containers.AbstractSingletonContainer.<init>(AbstractSingletonContainer.java:127)
        at com.sun.ejb.containers.CMCSingletonContainer.<init>(CMCSingletonContainer.java:76)
        at com.sun.ejb.containers.SingletonContainerFactory.createContainer(SingletonContainerFactory.ja                                                     va:68)
        at org.glassfish.ejb.startup.EjbApplication.loadContainers(EjbApplication.java:225)
        ... 24 more
Caused by: java.lang.ClassNotFoundException: fish.payara.nucleus.requesttracing.RequestTracingService
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 29 more

ago 23, 2016 6:20:14 PM com.sun.enterprise.v3.server.ApplicationLifecycle deploy
GRAVE: Exception during lifecycle processing
java.lang.RuntimeException: EJB Container initialization error
        at org.glassfish.ejb.startup.EjbApplication.loadContainers(EjbApplication.java:237)
        at org.glassfish.ejb.startup.EjbDeployer.load(EjbDeployer.java:290)
        at org.glassfish.ejb.startup.EjbDeployer.load(EjbDeployer.java:100)
        at org.glassfish.internal.data.ModuleInfo.load(ModuleInfo.java:206)
        at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:314)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:497)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:220)
        at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:487)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:539)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:535)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:534)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:565)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:557)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:556)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1464)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:109)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1846)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1722)
        at com.sun.enterprise.admin.cli.embeddable.DeployerImpl.deploy(DeployerImpl.java:134)
        at br.com.bematech.hubprinter.bootstrap.Server.deploy(Server.java:107)
        at br.com.bematech.hubprinter.bootstrap.Server.main(Server.java:127)
Caused by: java.lang.NoClassDefFoundError: fish/payara/nucleus/requesttracing/RequestTracingService
        at com.sun.ejb.containers.BaseContainer.<init>(BaseContainer.java:537)
        at com.sun.ejb.containers.AbstractSingletonContainer.<init>(AbstractSingletonContainer.java:127)
        at com.sun.ejb.containers.CMCSingletonContainer.<init>(CMCSingletonContainer.java:76)
        at com.sun.ejb.containers.SingletonContainerFactory.createContainer(SingletonContainerFactory.ja                                                     va:68)
        at org.glassfish.ejb.startup.EjbApplication.loadContainers(EjbApplication.java:225)
        ... 24 more
Caused by: java.lang.ClassNotFoundException: fish.payara.nucleus.requesttracing.RequestTracingService
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 29 more

ago 23, 2016 6:20:15 PM org.glassfish.api.ActionReport failure
GRAVE: Exception while loading the app
ago 23, 2016 6:20:15 PM org.glassfish.deployment.admin.DeployCommand execute
GRAVE: Exception while loading the app : EJB Container initialization error
java.lang.NoClassDefFoundError: fish/payara/nucleus/requesttracing/RequestTracingService
        at com.sun.ejb.containers.BaseContainer.<init>(BaseContainer.java:537)
        at com.sun.ejb.containers.AbstractSingletonContainer.<init>(AbstractSingletonContainer.java:127)
        at com.sun.ejb.containers.CMCSingletonContainer.<init>(CMCSingletonContainer.java:76)
        at com.sun.ejb.containers.SingletonContainerFactory.createContainer(SingletonContainerFactory.ja                                                     va:68)
        at org.glassfish.ejb.startup.EjbApplication.loadContainers(EjbApplication.java:225)
        at org.glassfish.ejb.startup.EjbDeployer.load(EjbDeployer.java:290)
        at org.glassfish.ejb.startup.EjbDeployer.load(EjbDeployer.java:100)
        at org.glassfish.internal.data.ModuleInfo.load(ModuleInfo.java:206)
        at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:314)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:497)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:220)
        at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:487)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:539)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:535)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:534)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:565)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:557)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:556)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1464)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:109)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1846)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1722)
        at com.sun.enterprise.admin.cli.embeddable.DeployerImpl.deploy(DeployerImpl.java:134)
        at br.com.bematech.hubprinter.bootstrap.Server.deploy(Server.java:107)
        at br.com.bematech.hubprinter.bootstrap.Server.main(Server.java:127)
Caused by: java.lang.ClassNotFoundException: fish.payara.nucleus.requesttracing.RequestTracingService
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 29 more

23-08-2016 18:20:15,374  INFO [main] (Server.java:31) - Watcher Service registered for dir: /home/servic                                                     es/settings
^C
root@hubprn1:/home/services/settings#
root@hubprn1:/home/services/settings#
root@hubprn1:/home/services/settings#
root@hubprn1:/home/services/settings# clear
root@hubprn1:/home/services/settings# killall java
root@hubprn1:/home/services/settings# killall java
java: nenhum processo localizado
root@hubprn1:/home/services/settings# clear
root@hubprn1:/home/services/settings# ./debug.sh
root@hubprn1:/home/services/settings# Listening for transport dt_socket at address: 8000
23-08-2016 18:23:03,725  INFO [main] (Server.java:119) -
#===== Initializing Printer Settings Embedded Server =====#

ago 23, 2016 6:23:34 PM org.jvnet.hk2.config.ConfigParser handleElement
GRAVE: Ignoring unrecognized element jms-service at Line number = 101
Column number = 72
System Id = null
Public Id = null
Location Uri= null
CharacterOffset = 5783

ago 23, 2016 6:23:37 PM org.glassfish.security.services.impl.authorization.AuthorizationServiceImpl init                                                     ialize
INFORMAÇÕES: Authorization Service has successfully initialized.
log4j:WARN No appenders could be found for logger (org.jboss.logging).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
ago 23, 2016 6:23:46 PM com.sun.enterprise.v3.services.impl.GrizzlyService createNetworkProxy
INFORMAÇÕES: Network listener https-listener on port 0 disabled per domain.xml
ago 23, 2016 6:23:46 PM com.sun.enterprise.v3.admin.adapter.AdminEndpointDecider setGuiContextRoot
INFORMAÇÕES: Admin Console Adapter: context root: /admin
ago 23, 2016 6:23:46 PM com.sun.enterprise.v3.admin.adapter.AdminEndpointDecider setGuiContextRoot
INFORMAÇÕES: Admin Console Adapter: context root: /admin
ago 23, 2016 6:23:46 PM com.sun.enterprise.v3.admin.adapter.AdminEndpointDecider setGuiContextRoot
INFORMAÇÕES: Admin Console Adapter: context root: /admin
ago 23, 2016 6:23:52 PM com.sun.enterprise.v3.server.AppServerStartup postStartupJob
INFORMAÇÕES: Undefined Product Name - define product and version info in config/branding  0.0.0.0.0  (0)                                                      startup time : Embedded (13.268ms), startup services(10.445ms), total(23.713ms)
ago 23, 2016 6:23:52 PM com.sun.enterprise.v3.server.AppServerStartup postStartupJob
GRAVE: Shutting down server due to startup exception
java.lang.NoClassDefFoundError: fish/payara/nucleus/requesttracing/RequestTracingService
        at com.sun.enterprise.v3.services.impl.ContainerMapper.<init>(ContainerMapper.java:107)
        at com.sun.enterprise.v3.services.impl.GlassfishNetworkListener.configureHttpProtocol(GlassfishN                                                     etworkListener.java:166)
        at org.glassfish.grizzly.config.GenericGrizzlyListener.configureSubProtocol(GenericGrizzlyListen                                                     er.java:420)
        at org.glassfish.grizzly.config.GenericGrizzlyListener.configureProtocol(GenericGrizzlyListener.                                                     java:409)
        at org.glassfish.grizzly.config.GenericGrizzlyListener.configure(GenericGrizzlyListener.java:305                                                     )
        at com.sun.enterprise.v3.services.impl.GrizzlyProxy.initialize(GrizzlyProxy.java:125)
        at com.sun.enterprise.v3.services.impl.GrizzlyService.createNetworkProxy(GrizzlyService.java:539                                                     )
        at com.sun.enterprise.v3.services.impl.GrizzlyService.postConstruct(GrizzlyService.java:490)
        at org.jvnet.hk2.internal.ClazzCreator.postConstructMe(ClazzCreator.java:326)
        at org.jvnet.hk2.internal.ClazzCreator.create(ClazzCreator.java:374)
        at org.jvnet.hk2.internal.SystemDescriptor.create(SystemDescriptor.java:471)
        at org.glassfish.hk2.runlevel.internal.AsyncRunLevelContext.findOrCreate(AsyncRunLevelContext.ja                                                     va:228)
        at org.glassfish.hk2.runlevel.RunLevelContext.findOrCreate(RunLevelContext.java:85)
        at org.jvnet.hk2.internal.Utilities.createService(Utilities.java:2022)
        at org.jvnet.hk2.internal.ServiceHandleImpl.getService(ServiceHandleImpl.java:114)
        at org.jvnet.hk2.internal.ServiceHandleImpl.getService(ServiceHandleImpl.java:88)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.oneJob(CurrentTaskFuture.ja                                                     va:1213)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.run(CurrentTaskFuture.java:                                                     1144)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$UpOneLevel.run(CurrentTaskFuture.java:7                                                     62)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$UpAllTheWay.go(CurrentTaskFuture.java:5                                                     61)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$UpAllTheWay.access$100(CurrentTaskFutur                                                     e.java:443)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture.go(CurrentTaskFuture.java:154)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture.go(CurrentTaskFuture.java:149)
        at org.glassfish.hk2.runlevel.internal.AsyncRunLevelContext.proceedTo(AsyncRunLevelContext.java:                                                     436)
        at org.glassfish.hk2.runlevel.internal.RunLevelControllerImpl.proceedTo(RunLevelControllerImpl.j                                                     ava:71)
        at com.sun.enterprise.v3.server.AppServerStartup.proceedTo(AppServerStartup.java:534)
        at com.sun.enterprise.v3.server.AppServerStartup.run(AppServerStartup.java:331)
        at com.sun.enterprise.v3.server.AppServerStartup.doStart(AppServerStartup.java:228)
        at com.sun.enterprise.v3.server.AppServerStartup.start(AppServerStartup.java:219)
        at com.sun.enterprise.glassfish.bootstrap.GlassFishImpl.start(GlassFishImpl.java:79)
        at br.com.bematech.hubprinter.bootstrap.Server.main(Server.java:124)
Caused by: java.lang.ClassNotFoundException: fish.payara.nucleus.requesttracing.RequestTracingService
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 31 more

ago 23, 2016 6:23:52 PM com.sun.enterprise.v3.admin.StopServer doExecute
INFORMAÇÕES: Server shutdown initiated
23-08-2016 18:23:52,870  INFO [main] (Server.java:106) - Deploying application
ago 23, 2016 6:24:03 PM com.sun.enterprise.security.SecurityLifecycle <init>
INFORMAÇÕES: Java security manager is disabled.
ago 23, 2016 6:24:03 PM com.sun.enterprise.security.SecurityLifecycle onInitialization
INFORMAÇÕES: Entering Security Startup Service.
ago 23, 2016 6:24:03 PM com.sun.enterprise.security.PolicyLoader loadPolicy
INFORMAÇÕES: Loading policy provider com.sun.enterprise.security.jacc.provider.SimplePolicyProvider.
ago 23, 2016 6:24:03 PM com.sun.enterprise.security.auth.realm.Realm doInstantiate
INFORMAÇÕES: Realm [admin-realm] of classtype [com.sun.enterprise.security.auth.realm.file.FileRealm] su                                                     ccessfully created.
ago 23, 2016 6:24:03 PM com.sun.enterprise.security.auth.realm.Realm doInstantiate
INFORMAÇÕES: Realm [file] of classtype [com.sun.enterprise.security.auth.realm.file.FileRealm] successfu                                                     lly created.
ago 23, 2016 6:24:03 PM com.sun.enterprise.security.auth.realm.Realm doInstantiate
INFORMAÇÕES: Realm [certificate] of classtype [com.sun.enterprise.security.auth.realm.certificate.Certif                                                     icateRealm] successfully created.
ago 23, 2016 6:24:03 PM com.sun.enterprise.security.SecurityLifecycle onInitialization
INFORMAÇÕES: Security Service(s) started successfully.
ago 23, 2016 6:24:05 PM com.sun.enterprise.web.WebContainer createHttpListener
INFORMAÇÕES: Created HTTP listener http-listener on host/port 0.0.0.0:80
ago 23, 2016 6:24:06 PM com.sun.enterprise.web.VirtualServer addProbes
GRAVE: Error adding HttpProbes. NetworkListener http-listeners GrizzlyProxy is NULL
ago 23, 2016 6:24:06 PM com.sun.enterprise.web.VirtualServer addProbes
GRAVE: Error adding HttpProbes. NetworkListener https-listeners GrizzlyProxy is NULL
ago 23, 2016 6:24:06 PM com.sun.enterprise.web.WebContainer createHosts
INFORMAÇÕES: Created virtual server server
ago 23, 2016 6:24:10 PM org.apache.catalina.realm.JAASRealm setContainer
INFORMAÇÕES: Setting JAAS app name glassfish-web
ago 23, 2016 6:24:10 PM com.sun.enterprise.web.WebContainer loadSystemDefaultWebModules
INFORMAÇÕES: Virtual server server loaded default web module
ago 23, 2016 6:24:11 PM org.apache.catalina.core.ContainerBase startChildren
GRAVE: Container StandardEngine[glassfish-web].StandardHost[server].StandardContext[] has not been start                                                     ed
org.apache.catalina.LifecycleException: org.apache.catalina.LifecycleException: java.lang.NoClassDefFoun                                                     dError: fish/payara/nucleus/requesttracing/RequestTracingService
        at org.apache.catalina.core.StandardContext.start(StandardContext.java:5926)
        at com.sun.enterprise.web.WebModule.start(WebModule.java:692)
        at org.apache.catalina.core.ContainerBase.startChildren(ContainerBase.java:1593)
        at org.apache.catalina.core.ContainerBase.start(ContainerBase.java:1290)
        at org.apache.catalina.core.StandardHost.start(StandardHost.java:982)
        at org.apache.catalina.core.ContainerBase.startChildren(ContainerBase.java:1593)
        at org.apache.catalina.core.ContainerBase.start(ContainerBase.java:1290)
        at org.apache.catalina.core.StandardEngine.start(StandardEngine.java:402)
        at org.apache.catalina.startup.Embedded.start(Embedded.java:995)
        at com.sun.enterprise.web.WebContainer.postConstruct(WebContainer.java:867)
        at org.jvnet.hk2.internal.ClazzCreator.postConstructMe(ClazzCreator.java:326)
        at org.jvnet.hk2.internal.ClazzCreator.create(ClazzCreator.java:374)
        at org.jvnet.hk2.internal.SystemDescriptor.create(SystemDescriptor.java:471)
        at org.jvnet.hk2.internal.SingletonContext$1.compute(SingletonContext.java:83)
        at org.jvnet.hk2.internal.SingletonContext$1.compute(SingletonContext.java:71)
        at org.glassfish.hk2.utilities.cache.Cache$OriginThreadAwareFuture$1.call(Cache.java:97)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at org.glassfish.hk2.utilities.cache.Cache$OriginThreadAwareFuture.run(Cache.java:154)
        at org.glassfish.hk2.utilities.cache.Cache.compute(Cache.java:199)
        at org.jvnet.hk2.internal.SingletonContext.findOrCreate(SingletonContext.java:122)
        at org.jvnet.hk2.internal.Utilities.createService(Utilities.java:2022)
        at org.jvnet.hk2.internal.ServiceHandleImpl.getService(ServiceHandleImpl.java:114)
        at org.jvnet.hk2.internal.ServiceHandleImpl.getService(ServiceHandleImpl.java:88)
        at org.glassfish.internal.data.EngineInfo.getContainer(EngineInfo.java:86)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.startContainers(ApplicationLifecycle.java:1                                                     012)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.setupContainerInfos(ApplicationLifecycle.ja                                                     va:718)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:379)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:220)
        at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:487)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:539)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:535)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:534)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:565)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:557)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:556)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1464)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:109)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1846)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1722)
        at com.sun.enterprise.admin.cli.embeddable.DeployerImpl.deploy(DeployerImpl.java:134)
        at br.com.bematech.hubprinter.bootstrap.Server.deploy(Server.java:107)
        at br.com.bematech.hubprinter.bootstrap.Server.main(Server.java:127)
Caused by: org.apache.catalina.LifecycleException: java.lang.NoClassDefFoundError: fish/payara/nucleus/r                                                     equesttracing/RequestTracingService
        at com.sun.enterprise.web.WebModuleContextConfig.start(WebModuleContextConfig.java:279)
        at org.apache.catalina.startup.ContextConfig.lifecycleEvent(ContextConfig.java:486)
        at org.apache.catalina.util.LifecycleSupport.fireLifecycleEvent(LifecycleSupport.java:163)
        at org.apache.catalina.core.StandardContext.start(StandardContext.java:5923)
        ... 45 more
Caused by: java.lang.NoClassDefFoundError: fish/payara/nucleus/requesttracing/RequestTracingService
        at org.apache.catalina.core.StandardWrapper.<init>(StandardWrapper.java:225)
        at org.apache.catalina.core.StandardContext.createWrapper(StandardContext.java:4220)
        at com.sun.enterprise.web.pwc.PwcWebModule.createWrapper(PwcWebModule.java:259)
        at com.sun.enterprise.web.TomcatDeploymentConfig.configureStandardContext(TomcatDeploymentConfig                                                     .java:402)
        at com.sun.enterprise.web.TomcatDeploymentConfig.configureWebModule(TomcatDeploymentConfig.java:                                                     108)
        at com.sun.enterprise.web.WebModuleContextConfig.start(WebModuleContextConfig.java:264)
        ... 48 more
Caused by: java.lang.ClassNotFoundException: fish.payara.nucleus.requesttracing.RequestTracingService
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 54 more

ago 23, 2016 6:24:16 PM org.glassfish.internal.data.ModuleInfo load
GRAVE: Exception while invoking class org.glassfish.ejb.startup.EjbDeployer load method
java.lang.RuntimeException: EJB Container initialization error
        at org.glassfish.ejb.startup.EjbApplication.loadContainers(EjbApplication.java:237)
        at org.glassfish.ejb.startup.EjbDeployer.load(EjbDeployer.java:290)
        at org.glassfish.ejb.startup.EjbDeployer.load(EjbDeployer.java:100)
        at org.glassfish.internal.data.ModuleInfo.load(ModuleInfo.java:206)
        at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:314)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:497)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:220)
        at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:487)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:539)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:535)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:534)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:565)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:557)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:556)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1464)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:109)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1846)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1722)
        at com.sun.enterprise.admin.cli.embeddable.DeployerImpl.deploy(DeployerImpl.java:134)
        at br.com.bematech.hubprinter.bootstrap.Server.deploy(Server.java:107)
        at br.com.bematech.hubprinter.bootstrap.Server.main(Server.java:127)
Caused by: java.lang.NoClassDefFoundError: fish/payara/nucleus/requesttracing/RequestTracingService
        at com.sun.ejb.containers.BaseContainer.<init>(BaseContainer.java:537)
        at com.sun.ejb.containers.StatelessSessionContainer.<init>(StatelessSessionContainer.java:143)
        at com.sun.ejb.containers.StatelessSessionContainer.<init>(StatelessSessionContainer.java:137)
        at com.sun.ejb.containers.StatelessContainerFactory.createContainer(StatelessContainerFactory.ja                                                     va:61)
        at org.glassfish.ejb.startup.EjbApplication.loadContainers(EjbApplication.java:225)
        ... 24 more
Caused by: java.lang.ClassNotFoundException: fish.payara.nucleus.requesttracing.RequestTracingService
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 29 more

ago 23, 2016 6:24:16 PM com.sun.enterprise.v3.server.ApplicationLifecycle deploy
GRAVE: Exception during lifecycle processing
java.lang.RuntimeException: EJB Container initialization error
        at org.glassfish.ejb.startup.EjbApplication.loadContainers(EjbApplication.java:237)
        at org.glassfish.ejb.startup.EjbDeployer.load(EjbDeployer.java:290)
        at org.glassfish.ejb.startup.EjbDeployer.load(EjbDeployer.java:100)
        at org.glassfish.internal.data.ModuleInfo.load(ModuleInfo.java:206)
        at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:314)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:497)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:220)
        at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:487)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:539)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:535)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:534)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:565)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:557)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:556)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1464)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:109)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1846)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1722)
        at com.sun.enterprise.admin.cli.embeddable.DeployerImpl.deploy(DeployerImpl.java:134)
        at br.com.bematech.hubprinter.bootstrap.Server.deploy(Server.java:107)
        at br.com.bematech.hubprinter.bootstrap.Server.main(Server.java:127)
Caused by: java.lang.NoClassDefFoundError: fish/payara/nucleus/requesttracing/RequestTracingService
        at com.sun.ejb.containers.BaseContainer.<init>(BaseContainer.java:537)
        at com.sun.ejb.containers.StatelessSessionContainer.<init>(StatelessSessionContainer.java:143)
        at com.sun.ejb.containers.StatelessSessionContainer.<init>(StatelessSessionContainer.java:137)
        at com.sun.ejb.containers.StatelessContainerFactory.createContainer(StatelessContainerFactory.ja                                                     va:61)
        at org.glassfish.ejb.startup.EjbApplication.loadContainers(EjbApplication.java:225)
        ... 24 more
Caused by: java.lang.ClassNotFoundException: fish.payara.nucleus.requesttracing.RequestTracingService
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 29 more

ago 23, 2016 6:24:17 PM org.glassfish.api.ActionReport failure
GRAVE: Exception while loading the app
ago 23, 2016 6:24:17 PM org.glassfish.deployment.admin.DeployCommand execute
GRAVE: Exception while loading the app : EJB Container initialization error
java.lang.NoClassDefFoundError: fish/payara/nucleus/requesttracing/RequestTracingService
        at com.sun.ejb.containers.BaseContainer.<init>(BaseContainer.java:537)
        at com.sun.ejb.containers.StatelessSessionContainer.<init>(StatelessSessionContainer.java:143)
        at com.sun.ejb.containers.StatelessSessionContainer.<init>(StatelessSessionContainer.java:137)
        at com.sun.ejb.containers.StatelessContainerFactory.createContainer(StatelessContainerFactory.ja                                                     va:61)
        at org.glassfish.ejb.startup.EjbApplication.loadContainers(EjbApplication.java:225)
        at org.glassfish.ejb.startup.EjbDeployer.load(EjbDeployer.java:290)
        at org.glassfish.ejb.startup.EjbDeployer.load(EjbDeployer.java:100)
        at org.glassfish.internal.data.ModuleInfo.load(ModuleInfo.java:206)
        at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:314)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:497)
        at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:220)
        at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:487)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:539)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:535)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:534)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:565)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:557)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:360)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:556)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1464)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:109)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1846)
        at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java                                                     :1722)
        at com.sun.enterprise.admin.cli.embeddable.DeployerImpl.deploy(DeployerImpl.java:134)
        at br.com.bematech.hubprinter.bootstrap.Server.deploy(Server.java:107)
        at br.com.bematech.hubprinter.bootstrap.Server.main(Server.java:127)
Caused by: java.lang.ClassNotFoundException: fish.payara.nucleus.requesttracing.RequestTracingService
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 29 more

23-08-2016 18:24:17,802  INFO [main] (Server.java:31) - Watcher Service registered for dir: /home/services/settings
```

**How I'm calling it:**
`java -Xdebug -agentlib:jdwp=transport=dt_socket,address=8000,server=y,suspend=n -jar /home/services/settings/printer-settings-embedded.jar -profile compact1 &`

**Main code using the container:**
```
public static void main( String[] args ) {
    	try 
        {
        	LOG.info("\n#===== Initializing Printer Settings Embedded Server =====#\n");
			GlassFishRuntime runtime = GlassFishRuntime.bootstrap();
	        GlassFishProperties gfproperties = new GlassFishProperties();
	        gfproperties.setPort("http-listener", 80);
	        gf = runtime.newGlassFish(gfproperties);
	        gf.start();
	        
	        //Deploy the application
			deploy();
			
			//setup the watcher
			setupWatcher();			
        }
        
        catch (GlassFishException ex) 
        {
			LOG.fatal("Could not start payara-embedded-web: " + ex.getMessage());
			LOG.fatal("location: " + ex.getLocalizedMessage());
			LOG.fatal("cause: " + ex.getCause());
			LOG.fatal("exception: " + ex);
        }
    }

private static void deploy() {
		try {
			LOG.info("Deploying application");
			gf.getDeployer().deploy(new File("/home/services/settings/deploy"), "--contextroot=/");
		} catch (GlassFishException ex) {
			LOG.error("Error on function deploy(): " + ex.getMessage());
			LOG.error("location: " + ex.getLocalizedMessage());
			LOG.error("cause: " + ex.getCause());
			LOG.error("exception: " + ex);
		}
	}
```

--- a/appserver/extras/embedded/web/pom.xml
+++ b/appserver/extras/embedded/web/pom.xml
@@ -271,6 +271,15 @@
 	    <type>zip</type>
 	    <optional>true</optional>
 	</dependency>
+
+        <!-- Request Tracing Package -->
+        <dependency>
+           <groupId>fish.payara.packager</groupId>
+           <artifactId>requesttracing-package</artifactId>
+           <version>${project.version}</version>
+           <type>zip</type>
+           <optional>true</optional>
+       </dependency>
     </dependencies>
 
     <profiles>

